eve/main.yml

```diff
@@ -39,3 +41,34 @@ stages:
             npm run flow &&
             npm run lint
           haltOnFailure: false
+  post-merge:
+    worker:
+      type: local
+    steps:
+      - TriggerStages:
+          name: Trigger storybook_deploy
+          stage_names:
+            - storybook_deploy
+          haltOnFailure: true
+  storybook_deploy:
```

Comment:
nit: I find it weird to use `-` and `_` as a seperator to name things, either use `-` everywhere or `_`

but this is just a personal thought, feel free to do whatever.